projects/i18n/src/modules/i18n/get-lib-string-for-locale.ts

```diff
@@ -11,11 +11,11 @@ import {
   },
   preferredLocale: string,
   name: string
-): string {
+): string | undefined {
```

Comment:
This function is internal and only exists in beta, so we can introduce this "breaking" change.